addons/medical/config.cpp

```diff
@@ -1,3 +1,7 @@
+#if __has_include("\z\ace\addons\nomedical\script_component.hpp")
```

Comment:
Does this even work when ACE is binned?

Reply:
no, medical components must be unbinned

Reply:
That's pretty bad.

Reply:
we're talking about configs, not models 
the difference between loading a binned vs non-binned config is probably a milisecond

Reply:
The bigger issue is that hemtt does not support this yet. so using this would break the build tools